lua/autorun/server/bounties.lua

```diff
@@ -0,0 +1,72 @@
+
+hook.Add("PlayerDeath","BountyDeath",function(ply,infl,atk)
+	local bounty = GetPlayerBounty(ply)
+	if bounty > 0 and ply != atk and type(atk) == "Player" then
+		SetPlayerBounty(ply,0)
+		atk:PS_GivePoints(bounty)
+		BotSayGlobal("[edgy]"..atk:Nick().." [fbc]has claimed [gold]"..ply:Nick().."'s [fbc]bounty of [rainbow]"..bounty.." [fbc]points!")
+	end
+end)
+
+function GetPlayerBounty(ply)
+	if ply.bounty == nil then
+		ply.bounty = tonumber(ply:GetPData("bounty",0))
+	end
+	return ply.bounty
+end
+
+function SetPlayerBounty(ply,bounty)
+	ply:SetPData("bounty",bounty)
+	ply.bounty = bounty
+end
+
+function AddBounty(ply,target,amount)
+	if type(ply) != "Player" or type(target) != "Player" or type(amount) != "number" then return end
+	amount = amount > 0 and amount or 0
+	if ply:PS_HasPoints(amount) then
+		SetPlayerBounty(target,GetPlayerBounty(target) + amount)
+		ply:PS_TakePoints(amount)
+		BotSayGlobal("[fbc]"..target:Nick().."'s bounty is now [rainbow]"..GetPlayerBounty(target) + amount.." [fbc]points")
+	else
+		ply:ChatPrint("[red]You don't have enough points")
+	end
+end
+
+RegisterChatCommand({'bounty'},function(ply,arg)
+	local t = string.Explode(" ",arg)
+	local to = PlyCount(t[1])
+	if type(to[1]) == "Player" and not to[2]>1 then
+		local bounty = GetPlayerBounty(to)
+		local p = tonumber(t[2])
+		if type(p) == "number" then
+			if p > 1000 then
+				AddBounty(ply,to,p)
+			else
+				ply:ChatPrint("[red]You must add a minimum of 1000 points to the bounty")
+			end
+		elseif bounty > 0 then
+			BotSayGlobal(to:Nick().."'s bounty is [rainbow]"..bounty.." [fbc]points")
```

Comment:
did you mean for this to be global? if so i guess thats fine because of the throttle thing